vehicle/mkv/software/throttle/throttle.c

```diff
@@ -47,44 +54,90 @@ int main(void) {
     gpio_enable_interrupt(SS_IS);
     gpio_enable_interrupt(SS_BOTS);
 
-	// Initialize interrupts
-	sei();
+    // Initialize interrupts
+    sei();
+
+    // Initialize motor controller
+    CANMotorController[0] = 0;
+    CANMotorController[1] = 0;
+    CANMotorController[2] = 0;
+    CANMotorController[3] = 0;
+    CANMotorController[4] = 1; // anticlockwise
+    CANMotorController[5] = 0;
+    CANMotorController[6] = 0;
+    CANMotorController[7] = 0;
```

Comment:
Don't need to do all of this since you initialize the variable above. Instead, just do

```c
can_motor_controller[4] = 1;
```

And leave out all of the zeroes.